FY2017/FY2017.beancount

```diff
@@ -0,0 +1,8 @@
+2017-01-01 open Assets:Operations:PNC USD
+
+; in order to achieve opening balance on 2017-01-01, add difference between 2883.99 and the latest balance of PNC (currently 0)
+2017-01-01 * "Opening Balance"
+    Assets:Operations:PNC           2883.99 USD
+    Equity:Owners:Chad-Whitacre    -2883.99 USD
```

Comment:
@whit537 does this make sense? This is to cover the change to PNC account before 2017.